tests/test_mprester.py

```diff
@@ -26,11 +28,14 @@
 from mp_api.client import MPRester
 from mp_api.client.core.settings import MAPIClientSettings
 
-os.environ["MP_API_KEY"] = "test"
-
 
 @pytest.fixture()
 def mpr():
+    # Only mock the API key in GitHub CI environment,
+    # otherwise people cannot run tests locally as the key is invalid
+    if os.getenv("GITHUB_ACTIONS") is not None:
```

Comment:
Need confirm: would this test suite be intended to run on other CI env other than GitHub (GitLab or some self-hosted runner?)